source/index.rst

```diff
@@ -18,6 +18,30 @@ With `Inrupt PodBrowser`_, you can
 
 - Manage access controls to your data.
 
+Browser Support
+===============
+
+You can use `PodBrowser <https://podbrowser.inrupt.com>`_ with the
```

Comment:
In theory you can "use" PodBrowser with a wide range of browsers. The statement should more relate to the browsers that we support for use with PodBrowser.

Reply:
Updated as PodBrowser  supports the following browsers: